Core/tests/ExpressionParser2.cpp

```diff
@@ -1519,6 +1519,14 @@ TEST_CASE("ExpressionParser2", "[common][events]") {
       node->Visit(validator);
       REQUIRE(validator.GetFatalErrors().size() == 0);
     }
+    {
```

Comment:
This should probably be in a different test "Variable with bracket accessor and operators".

Add checks for *, -, /

If you can also make the test to check what is code generated, that would be safer.

Reply:
Also add tests for "Variable with nested bracket accessor and operator" which does something like:

`MySceneStructureVariable[MySceneStructureVariable2["MyChild"]] * 2` (with all operators) 
`MySceneStructureVariable[MySceneStructureVariable2["MyChild"] * 2] * 3` (with all operators)

both for the validator and for code generation.